framework/widgets/ActiveField.php

```diff
@@ -697,7 +697,7 @@ public function widget($class, $config = [])
         $config['model'] = $this->model;
         $config['attribute'] = $this->attribute;
         $config['view'] = $this->form->getView();
-        if (in_array('yii\widgets\InputWidget', class_parents($class)) && isset($config['options'])) {
+        if (isset($config['options']) && isset(class_parents($class)['yii\widgets\InputWidget'])) {
```

Comment:
Are you sure it would work?

`class_parents($class)` gives you flat array, not keys.